rows.go

```diff
@@ -29,7 +29,6 @@ var (
 
 type snowflakeRows struct {
 	sc                  *snowflakeConn
-	RowType             []execResponseRowType
```

Comment:
The expected behavior is to remove this field. We never intentionally touched it, so we should align with upstream code.

This field was removed from upstream on Feb 10th: https://github.com/snowflakedb/gosnowflake/commit/9143c1f7011ec4acd4d1bfdac81b16cd1d18cc13#diff-f86d9238681ad0d9a7556edc0f847c2760a63f658dccea75c1d5c27b6b179badL32

However it seems we missed the upstream change in previous merge/rebase: https://github.com/observeinc/gosnowflake/blob/625c54493b7ccc298d4efd6b790b0135c5c9dd15/rows.go#L32

Let us fix that by removing it.